### DISCUSSION TITLE
ccf-validation-tools/Pancreas/comments/

### DISCUSSION THREAD

**Discussion by giscus** [hubmapconsortium/ccf-validation-tools#270] [General]
# ccf-validation-tools/Pancreas/comments/

https://hubmapconsortium.github.io/ccf-validation-tools/Pancreas/comments/

**Comment by emquardokus** [comment]
Pancreas_v1.3_DRAFT
https://docs.google.com/spreadsheets/d/1dAnjj6RMzIcaV0t_njMhVHtDlLkEDCqtLteY9YdF7iM/edit#gid=439021026

**Different labels**
 -[ ]Removed the trailing space after connective tissue: In row 45, the term UBERON:0002384 has different name/label in the source ontology. The name/label in the ASCT+B table is connective tissue _ and the one in the ontology is _connective tissue. For reference, the given name/label by SMEs is Interlobular connective tissue. Please correct it in the columns AS/N/LABEL or CT/N/LABEL in the ASCT+B table.

**Comment by emquardokus** [comment]
Pancreas_v1.3_DRAFT
https://docs.google.com/spreadsheets/d/1dAnjj6RMzIcaV0t_njMhVHtDlLkEDCqtLteY9YdF7iM/edit#gid=439021026

**Different labels**
FIXED: In row 19, the term CL:0002080 has different name/label in the source ontology. The name/label in the ASCT+B table is pancreatic centroacinar cell and the one in the ontology is pancreatic centro-acinar cell. For reference, the given name/label by SMEs is Centroacinar cell. Please correct it in the columns AS/N/LABEL or CT/N/LABEL in the ASCT+B table.

**Comment by aleixpuigb** [comment]
New term added in the last Uberon release (v2024-01-18):

Pancreas: 'neck of pancreas' (UBERON:8480032)
